.cypress/cypress/integration/merton.js

```diff
@@ -0,0 +1,28 @@
+it('loads the Merton FMS Pro front page', function() {
+    cy.visit('http://merton.localhost:3001/');
+    cy.contains('Merton Council');
+});
+it('does not allow typing >255 chars in additional question fields', function() {
+    //cy.server();
```

Comment:
If this line isn't required can it be removed instead of left commented out?

Reply:
🤦🏼